cargo-shuttle/src/lib.rs

```diff
@@ -952,7 +973,29 @@ impl Shuttle {
                 _ => CommandOutcome::Ok,
             })
         } else {
-            println!("Deployment has not entered the running state");
+            println!("{}", "Deployment has not entered the running state".red());
+            println!();
+            match last_state {
+                Some(shuttle_common::deployment::State::Stopped) => {
+                    println!("State: Stopped - Deployment was running, but has been stopped by the user.")
+                }
+                Some(shuttle_common::deployment::State::Completed) => {
+                    println!("State: Completed - Deployment was running, but stopped running all by itself.")
+                }
+                Some(shuttle_common::deployment::State::Unknown) => {
+                    println!("State: Unknown - This may be because deployment was in an unknown state. We never expect this state and entering this state should be considered a bug.")
+                }
+                Some(shuttle_common::deployment::State::Running) => {
+                    println!("State: Running - Deployment is running ie. its thread is active.")
```

Comment:
This should not be possible, and if it were to happen it would conflict with the default println at the top here (`Deployment has not entered the running state`). The reason it should not be possible is if we enter this branch, it's because the `client.get_service` call above did not find any running deployment for this project. 

I'm also wondering if we should rather than calling `get_service` and seeing if it finds a running deployment, we should call `get_deployment` with the deployment id of the deployment we started in this `deploy` call. This guarantees that we are checking the state of the deployment we started here, although the current method should also work because when a deployment is promoted to `Running` it should first stop any other running deployment. TLDR: consider this second paragraph optional, unless you have any thoughts on the matter! :smile: